config/samples/go_example_loadtest.yaml

```diff
@@ -27,6 +27,7 @@ spec:
     language: go
     name: '0'
     run:
+    - name: "main"
```

Comment:
Please make the fields alphabetical.

you should update the templates, run loadtest_examples.sh to generate the examples, then use loadtest_template.py to generate the templates again. Once you do that, the templates should not change (it is a round trip).

I would include all example updates in this PR.